src/avgScrabbleScoreFromFile.js

```diff
@@ -11,12 +11,17 @@ let avgScrabbleScore = require('./avgScrabbleScore');
  * sample-words.txt.
  */
 function avgScrabbleScoreFromFile(fileName) {
-  // This is your job. :)
+  words = fs.readFileSync(`./${fileName}`, 'utf8')
```

Comment:
I'd just have `readFileSync(fileName)`. Reading from the current directory is the default and the caller can decide if they want to make the path more complex. As it stands now, the caller would have to know this internal detail before figuring out what file name to pass in.